src/gbt.cpp

```diff
@@ -76,159 +52,118 @@ bool scanGBTPhasesLocal(localArgs *la, const uint32_t ohN, const uint32_t nReset
             // Check the VFAT status
             slowCtrlErrCntVFAT vfatErrs;
             for (uint32_t vfatN = 0; vfatN < oh::VFATS_PER_OH; vfatN++) {
-                // check SYNC_ERR_CNT
-                if (readReg(la, stdsprintf("GEM_AMC.OH_LINKS.OH%hu.VFAT%hu.SYNC_ERR_CNT", ohN, vfatN)) != 0){
+                syncErrCnt_regName << "GEM_AMC.OH_LINKS.OH" << ohN << ".VFAT" << vfatN << ".SYNC_ERR_CNT";
```

Comment:
Is the scan function working? The `std::stringstream`'s are not cleared between the different VFATs... Since no string formatting is required, a `std::string` would just work as well.